games/__init__.py

```diff
@@ -0,0 +1,5 @@
+# Sentinel for the first game in the series
+FIRST_GAME = object()
+
+# Sentinel object for the last game in the series
+LAST_GAME = object
```

Comment:
Should there be parentheses after 'object'?

Reply:
Yes, good catch

Reply:
Fixed in adad2ae